device.mk

```diff
@@ -280,11 +276,6 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.audio.low_latency.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/android.hardware.audio.low_latency.xml
 
-# Boot animation
-TARGET_SCREEN_HEIGHT := 1280
-TARGET_SCREEN_WIDTH := 720
-TARGET_BOOTANIMATION_HALF_RES := true
```

Comment:
isn't it worked?

Reply:
they only work in [lineage](https://github.com/LineageOS/android_vendor_lineage/blob/lineage-15.1/bootanimation/Android.mk)
`TARGET_BOOT_ANIMATION_RES` is for [AEX](https://github.com/AospExtended/platform_vendor_aosp/blob/8.1.x/config/bootanimation.mk)

Reply:
> [they](https://github.com/LineageOS/android_vendor_lineage/blob/lineage-15.1/bootanimation/Android.mk) only work in lineage
> `TARGET_BOOT_ANIMATION_RES` is for [AEX](https://github.com/AospExtended/platform_vendor_aosp/blob/8.1.x/config/bootanimation.mk)

Yup thats true

Reply:
okey then
merged